appveyor.yml

```diff
@@ -51,24 +51,27 @@ install:
 
     # Add path, activate `conda` and update conda.
     - cmd: call %CONDA_INSTALL_LOCN%\Scripts\activate.bat
-    - cmd: conda update --yes --quiet conda
+    - cmd: conda.exe update --yes --quiet conda
 
     - cmd: set PYTHONUNBUFFERED=1
 
     # Add our channels.
-    - cmd: conda config --set show_channel_urls true
-    - cmd: conda config --remove channels defaults
-    - cmd: conda config --add channels defaults
-    - cmd: conda config --add channels conda-forge
+    - cmd: conda.exe config --set show_channel_urls true
+    - cmd: conda.exe config --remove channels defaults
+    - cmd: conda.exe config --add channels defaults
+    - cmd: conda.exe config --add channels conda-forge
+    # Add the pycalphad upstream channels
+    - cmd: conda.exe config --add channels msys2
+    - cmd: conda.exe config --add channels pycalphad
```

Comment:
This is not the correct way to do this @bocklund. First `msys2` is part of `defaults`. Second we don’t rely on other channels in `conda-forge` with the exception of `defaults`. Ideally we want to reduce our dependency on `defaults` as well. Third one should avoid editing the CI files directly. Is it possible for you to correct this with `epsei` on Windows in `conda-forge`?

Reply:
Hi @jakirkham, sorry for the delayed reply.

`pycalphad` is a hard dependency of ESPEI, which requires `ipopt` as of `pycalphad>=0.6`. `ipopt` is not currently supported on Windows via conda-forge (https://github.com/conda-forge/ipopt-feedstock/pull/11). The obvious fix would be to resolve this issue so we can build and release `ipopt`, `cyipopt` and their dependencies for Windows on conda-forge.

If there's no way that this is permissible, then a possible workaround might be for ESPEI to be set to `noarch`, but I would really like to avoid that, if possible.